src/serializers.ts

```diff
@@ -1,14 +1,14 @@
+import { PayloadRepository, WebhookEvent } from '@octokit/webhooks'
 import bunyan from 'bunyan'
 import express from 'express'
-import { PayloadRepository } from './context'
 
 export const serializers: bunyan.StdSerializers = {
 
-  event: (event: any) => {
+  event: (event: WebhookEvent | any) => {
```

Comment:
I had to add that to appease TypeScript, otherwise it complained that the next line was an unnecessary conditional. It could probably be `string`, but I used `any` since this method will get called any time there's an `event` key in a log message, and we have no control over what that value could be.

```
context.log({event: 'a user can put anything here})
```

It'll _probably_ always be a string, but 🤷‍♂️ 

What do you think?

Reply:
Ah that's clearer to me now. I think we should leave it `any` then, since it would not be out of the question for an app to implement a type that's more than just a string.